plugins/ern_v0.34.0+/react-native-gesture-handler_v1.4.1+/README.md

```diff
@@ -0,0 +1,28 @@
+## [React Native Gesture Handler](https://github.com/kmagiera/react-native-gesture-handler) plugin.
+
+### Prerequisites
+[Electrode Native](https://github.com/electrode-io/electrode-native) >= 0.38.0
+
+### To add this plugion to your miniapp
+
+`ern add react-native-gesture-handler`
+
+## For Android
+
+Create a custom delegate(`ReactNativeGestureDelegate.java`) by extending `ElectrodeReactFragmentActivityDelegate.java` and override the following method.
```

Comment:
`ElectrodeReactFragmentActivityDelegate ` is deprecated. Please use ElectrodeNavigationActivityDelegate instead.